src/sentry/tsdb/base.py

```diff
@@ -226,6 +227,11 @@ def get_active_series(self, start=None, end=None, timestamp=None):
             rollups[rollup] = map(to_datetime, series)
         return rollups
 
+    def make_series(self, default, start, end=None, rollup=None):
+        f = default if isinstance(default, collections.Callable) else lambda timestamp: default
```

Comment:
autoformatter rewrites the lambda into a `def` statement and linter won't allow a `def` statement that shadows an argument assignment so this is a little kludgy but is prevents this from expanding into several unnecessary lines

Reply:
How does `isinstance(default, collections.Callable)` differ from `callable(default)` ?

Reply:
My understanding is they're functionally equivalent. (I chose this one because I forgot that the builtin `callable` exists when I wrote this 😬 )